client/src/components/map/legend/item/toolbar/index.tsx

```diff
@@ -36,22 +40,33 @@ export const LegendItemToolbar: React.FC<LegendItemToolbarProps> = ({
             >
               <Tooltip delayDuration={500}>
                 <PopoverTrigger asChild>
-                  <TooltipTrigger
-                    type="button"
-                    aria-label="Change layer opacity"
-                    className={cn({
-                      'pointer-events-none': popoverOpen,
-                    })}
-                  >
-                    <LegendItemButton Icon={Droplet} value={opacity} selected />
+                  <TooltipTrigger asChild>
```

Comment:
I found some differences with the design. I am not sure if you talked with Lidia to make this changes but please check them:
- The button icons should have only 8px gap between them
- We don't need that separator for the close button
- The legend should have a unit. I think we need to add this to the layer data
- The gradient or colors of the legend should go almost full width 
- We have less padding for this legend. Just 8px (p-2 instead of p-3)
<img width="387" alt="Screenshot 2023-10-11 at 11 10 26" src="https://github.com/Orcasa-Platform/orcasa/assets/9701591/26e85950-1594-4440-8890-3f68016d08a7">

Reply:
Also, the droplet icon is not exactly the same. I wrote a message about this on the project channel